ruby_on_rails/template.rb

```diff
@@ -0,0 +1,33 @@
+# replace rubocop-rails-omakase with renuocop
+
+gsub_file "Gemfile", /gem "rubocop-rails-omakase"/, "gem \"renuocop\""
+
+# override the content of .rubocop.yml
+create_file ".rubocop.yml", force: true do
+  <<~RUBOCOP
+    inherit_gem:
+      renuocop: config/base.yml
+  RUBOCOP
+end
+
+after_bundle do
+  run "bundle exec rubocop -A"
+end
+
+create_file "bin/fastcheck", force: true do
+  <<~FASTCHECK
+    #!/usr/bin/env bash
+    set -euo pipefail
+
+    if ! bundle exec rubocop -D -c .rubocop.yml --fail-fast
+    then
+      echo 'rubocop detected issues!'
+      bundle exec rubocop -A -D -c .rubocop.yml
```

Comment:
I'd like to also just do the risky move when running fastcheck: 
* it's most of the times ok
* we have great test coverage
* we must review manually anyway

Reply:
agreed, since you have to add them with git again, i think this is feasible. but maybe it's harder to see which ones where forced changes. i see pro's and cons for that